irepela/homework_8/every_person.py

```diff
@@ -0,0 +1,43 @@
+# Details:  https://py.checkio.org/mission/every-person-is-unique/share/1c6fd4b663f7e40fa15b699f17efab40/
+from datetime import datetime
+from math import floor
+
+
+class Person:
+
+    def __init__(self, first_name, last_name, birth_date, job, working_years, salary, country, city,
+                 gender='unknown'):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.birth_date = birth_date
+        self.job = job
+        self.working_years = working_years
+        self.salary = salary
+        self.country = country
+        self.city = city
+        self.gender = gender
+
+    def name(self):
+        return f"{self.first_name} {self.last_name}"
+
+    def age(self):
+        current_date = datetime.strptime("01.01.2018", "%d.%m.%Y")
+        birth_date = datetime.strptime(self.birth_date, "%d.%m.%Y")
+        age = floor((current_date - birth_date).days / 365)
```

Comment:
can you avoid division here, 
division is always about precision and binary data processing, so if you can avoid it, it is always better